builtin/nomad/jobspec/plugin.proto

```diff
@@ -12,11 +12,6 @@ message Deployment {
   opaqueany.Any resource_state = 3;
 }
 
-message Release {
-  string Url = 1;
-  opaqueany.Any resource_state = 2;
-}
```

Comment:
@catsby regarding backwards-compatibility, a waypoint.hcl file using the `nomad-jobspec-canary` releaser will work as-is with this change. However, I have not tested the path where a deployment was created with 0.8.x, and then attempted to be destroyed with this branch. I can try that out, but I'm okay with un-backporting this.

Reply:
Deleting the definition won't break anything. I presume (but don't see?) that nothing is using this message as a type in another message anymore.

With protobuf, changing the type of existing fields is the number 1 backward compat issue. Deleting types just means you can't decode them in the future (but since you deleted the type, it's assumed you don't need to decode them either).